src/crypto/rfc8439.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_CRYPTO_RFC8439_H
+#define BITCOIN_CRYPTO_RFC8439_H
+
+#include <crypto/poly1305.h>
+#include <span.h>
+
+#include <array>
+#include <cstddef>
+#include <vector>
+
+constexpr static size_t RFC8439_KEYLEN = 32;
```

Comment:
In 7a9d2fb8cf11eef8b6bd87ad1066df374bf3b620: These could be
```suggestion
constexpr static size_t RFC8439_KEYLEN{32};
```